### DISCUSSION TITLE
revalidateTag with regexp or wildcard

### DISCUSSION THREAD

**Discussion by omermecitoglu** [vercel/next.js#66864] [App Router] [answered]
I'm not sure if this is a good practice but I cache everything separately. like `user:1`, `user:2`, `user:3`

So when I want to revalidate all the users, I need to know all the ids and revalidate them one by one

It would be so good if we could revalidate tags with certain patterns like `/^user:.*$/` or `user:*`

**Comment by wwwdepot** [comment, marked as answer]
there is no need to reinvent the wheel

1. fetch with `tags: ["users", "user:1"]`
2. then `revalidateTag("users")`
